src/MarginBase.sol

```diff
@@ -329,54 +328,25 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
                                 COMMANDS
     //////////////////////////////////////////////////////////////*/
 
-    function _perpsV2Deposit(address _market, int256 _amount) internal {
-        if (_amount <= 0) {
-            revert InvalidMarginDelta();
-        } else if (_abs(_amount) > freeMargin()) {
-            revert InsufficientFreeMargin(freeMargin(), _abs(_amount));
-        } else {
+    function _perpsV2ModifyMargin(address _market, int256 _amount) internal {
+        if (_amount > 0) {
+            if (_abs(_amount) > freeMargin()) {
```

Comment:
Technically don't need `_abs`

Reply:
freeMargin() returns a uint

Reply:
I can just cast instead of calling abs tho

Reply:
Your code used the same conv technique which is why I followed:
```
if (_marginDelta > 0) {
            // ensure margin doesn't exceed max
            if (_abs(_marginDelta) > freeMargin()) {
                revert InsufficientFreeMargin(freeMargin(), _abs(_marginDelta));
            }
            committedMargin += _abs(_marginDelta);
        }
```